src/vs/workbench/contrib/positronNotebook/browser/PositronNotebookEditor.tsx

```diff
@@ -198,21 +217,32 @@ export class PositronNotebookEditor extends AbstractEditorWithViewState<INoteboo
 	protected override createEditor(parent: HTMLElement): void {
 
 		this._logService.debug(this._identifier, 'createEditor');
-		this._parentDiv = DOM.$('.positron-notebook-container');
-		// Make container focusable so it can maintain focus when empty
-		this._parentDiv.tabIndex = -1;
-		parent.appendChild(this._parentDiv);
-		this._parentDiv.style.display = 'relative';
+
+		// Create the top-level editor container
```

Comment:
@dhruvisompura @nstrayer another big'ish change is to introduce one more nesting to the DOM tree:

- `positron-notebook-editor` is what the context key service is scoped to and where we track focus, it's the parent node
    - `positron-notebook-container` is the React root for the notebook component - the fact that this is a React root is why I added another nesting, since this gets destroyed when the notebook instance is detached e.g. when changing the active editor
    - `positron-notebook-overlay-container` is a parent that overlay widgets (like the find widget) can append their own React roots to